starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,60 @@
+body {
+margin: 0px;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
+
+figure {
+margin: 10px;
+}
+
+.applePies {
```

Comment:
This is one of the things that is causing your webpage to be really wide and have a horizontal scroll.  The reason for this is that this element is a block element, and when you move it over on the page, it continues taking up all the width a block element normally takes up, but it is moved over so it continues taking up space off the page to the right.